kubernetes_deploy/staging/deployment.yml

```diff
@@ -55,3 +56,62 @@ spec:
             secretKeyRef:
               name: db
               key: port
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-queue-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: laa-legal-adviser-api-queue-worker
+        env: staging
+        service_area: laa-get-access
+        service_team: cla-fala
+    spec:
+      containers:
+      - image: "<to be set by deploy_to_kubernetes>"
+        name: app
+        args: ["/usr/local/bin/python", "/home/app/manage.py", "celery", "worker", "-A laalaa", "--concurrency=1", "--loglevel=INFO"]
+        env:
+        - name: LOG_LEVEL
+          value: INFO
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: secret-key
+              key: SECRET_KEY
+        - name: SENTRY_DSN
+          valueFrom:
+            secretKeyRef:
+              name: sentry
+              key: dsn
+        - name: DB_USERNAME
```

Comment:
_notes, not a blocker_
:memo: To capture the conversation we had earlier, we discussed

- re-configuring the terraform module so we can have a single `DB_URL` configuration instead of 5 variables
- using [`kustomize`](https://github.com/kubernetes-sigs/kustomize) to remove repetition

(I would favour the former)

Reply:
> re-configuring the terraform module so we can have a single DB_URL configuration instead of 5 variables

This is being looked at by @exonian in a separate issue